beacon_node/beacon_chain/src/custody_context.rs

```diff
@@ -36,6 +38,23 @@ struct ValidatorRegistrations {
 }
 
 impl ValidatorRegistrations {
```

Comment:
Maybe we can document here that Epoch 0 is being used to store the cli parameter that the node was started with?

Reply:
https://github.com/sigp/lighthouse/pull/8254/commits/9a35a81c54d75a4f9c0179135e90f6b2ec0de23f